Source/Renderer/UniformState.js

```diff
@@ -885,10 +906,7 @@ define([
         this._resolutionScale = canvas.width / canvas.clientWidth;
 
         var camera = frameState.camera;
-
-        setView(this, camera.viewMatrix);
-        setInverseView(this, camera.inverseViewMatrix);
-        setCamera(this, camera);
+        this.updateCamera(camera);
```

Comment:
I haven't looked at the rest of the code yet, but does it make sense to not call this here and require the user to call it, e.g.,
```
renderFrame
  uniformState.update()
  foreach camera
    uniformState.updateCamera()
```